src/dotnet-symbol/dotnet-symbol.csproj

```diff
@@ -1,23 +1,20 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project Sdk="RoslynTools.RepoToolset">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <RuntimeFrameworkVersion>2.1.0</RuntimeFrameworkVersion>
```

Comment:
Please remove the `RuntimeFrameworkVersion`. This is not a good idea to embed unless you have very good reasons for it.

Reply:
I had a good reason with I first wrote this. I don't remember now.  I guess I'll remove it and see what breaks (if anything).